NewsApp/NewsApp/Controller/ProfileController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ProfileController.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 26/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ProfileController: UITableViewController {
+    @IBOutlet private weak var button: UIButton!
+    @IBOutlet private weak var picker: UIPickerView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let country = UserDefaults().string(forKey: "Country") {
```

Comment:
Tu peux utiliser la même classe pour les constantes dans ce cas ici

Reply:
fix in "fix PR"